pkg/eval/math/math.go

```diff
@@ -76,6 +76,22 @@ import (
 // ▶ (float64 -1)
 // ```
 
+//elvdoc:fn cosh
+//
+// ```elvish
+// math:cosh $number
+// ```
+//
+// Computes the hyperbolic cosine of `$number` in units of radians (not degrees).
```

Comment:
My knowledge of hyperbolic trigonometric functions is a bit rusty, but IIRC they are unrelated to geometric angles, so the note about the unit does not apply.

Reply:
Yes, you're right. I haven't thought about (or used) these functions since high school.